README.md

```diff
@@ -4,53 +4,36 @@ A clean, simple, and unobtrusive ruby authentication solution.
 
 [![Gem Version][5]][6] [![Build Status][1]][2] [![Code Climate][7]][8] [![Dependency Status][3]][4]
 
-## Sponsors
-
-[![Timber Logging](http://res.cloudinary.com/timber/image/upload/v1490556810/pricing/sponsorship.png)](https://timber.io?utm_source=github&utm_medium=authlogic)
-
-[Tail Authlogic users](https://timber.io/docs/app/console/tail-a-user) in your logs!
```

Comment:
Please leave the sponsors section here, at the top of the page, for now. That's something we'd have to ask Ben about. Please put it in a separate PR and we can discuss it there.